int/hwsc-user-svc/proto/hwsc-user-svc.proto

```diff
@@ -0,0 +1,203 @@
+syntax="proto3";
+
+package hwscUserSvc;
+
+option go_package="hwscDocumentSvcPb";
+
+service UserService {
+    rpc GetStatus (UserRequest) returns (UserResponse) {}
+
```

Comment:
we need ListUsers rpc

Reply:
what does this do?

Reply:
List all the users, you need this for epic https://github.com/faraonc/hwsc-gateway-svc/issues/75